src/main/java/seedu/storemando/ui/ResultDisplay.java

```diff
@@ -25,4 +25,8 @@ public void setFeedbackToUser(String feedbackToUser) {
         resultDisplay.setText(feedbackToUser);
     }
 
+    public void welcomeMsg() {
+        resultDisplay.setText("Welcome to StoreMando!!!");
```

Comment:
My personal opinion is that the welcome message should just be "Welcome to StoreMando!". Just a suggestion to consider.